assets/js/responsive_tables.js

```diff
@@ -0,0 +1,20 @@
+// set data-title attribute on table cells
+function responsiveTables() {
+  const tables = document.getElementById("main-content-wrapper").getElementsByTagName("table")
+  for (table of tables) {
+    const headings = table.getElementsByTagName("th")
+    const tbody = table.getElementsByTagName("tbody")
+    if (tbody.length > 0){
```

Comment:
Is the `tbody` element included in every table by hugo?

Reply:
I thought about this and I think we can be safe assuming that Hugo will create valid tables in the same way every time, so it should be safe to assume that `tbody` exists.